api/src/main/java/edu/wpi/first/shuffleboard/api/sources/recording/MarkerImportance.java

```diff
@@ -0,0 +1,58 @@
+package edu.wpi.first.shuffleboard.api.sources.recording;
+
+/**
+ * The levels of importance of a marker.
+ */
+public enum MarkerImportance {
+
+
+  // Maintainer note: this enum is mirrored in WPILib. Adding or removing entries, or changing the name of any entry,
+  // requires a corresponding change to WPILib.
```

Comment:
Move to a common library?

Reply:
Not really feasible since it has to be done in both C++ and Java WPILib, and I don't think the enum will change much (or at all), so the overhead of maintaining another (_tiny_) library just for this is not worth it IMO